src/DetailsView/components/report-export-button.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { InsightsCommandButton } from 'common/components/controls/insights-command-button';
+import { NamedFC } from 'common/react/named-fc';
+import { IButton, IRefObject } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+export interface ReportExportButtonProps {
+    showReportExportDialog: () => void;
+    buttonRef?: IRefObject<IButton>;
```

Comment:
Should this be optional?

Reply:
It's not strictly needed, since all we do is pass it through to another optional prop in the OF button component, and it's only useful if the caller wants to set focus manually.

Reply:
Gotcha! didn't know the OF prop was optional.